scripts/deploy-backend.sh

```diff
@@ -62,6 +64,8 @@ set_cf_envs()
         cf_cmd="cf unset-env $CGAPPNAME_BACKEND $var_name ${!var_name}"
         $cf_cmd
         continue
+    elif [[ ("$var_name" =~ "STAGING_*") && ("$CF_SPACE" = "tanf-staging") ]]; then
```

Comment:
Should this be `("$CF_SPACE" == "tanf-staging")`?

Reply:
Initially thought the same thing but saw the single `=` used in other comparisons so went with what was used elsewhere.